packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,124 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+
+/**
+ * A Tooltip Custom HTML Element.
+ * Based on ARIA APG Tooltip Pattern {@link https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/ }
+ * @public
+ */
+export class Tooltip extends FASTElement {
+  /**
+   * The attached element internals
+   */
+  public elementInternals = this.attachInternals();
+
+  /**
+   * Set the delay for the tooltip
+   */
+  @attr
+  public delay?: number;
+
+  /**
+   * The default delay for the tooltip
+   * @internal
+   */
+  private defaultDelay: number = 250;
+
+  /**
+   * Set the positioning of the tooltip
+   */
+  @attr
+  public positioning?: string;
+
+  /**
+   * The id of the anchor element for the tooltip
+   */
+  @attr
+  public anchor: string = '';
+
+  /**
+   * Reference to the anchor element
+   * @internal
+   */
+  private get anchorElement(): HTMLElement | null {
+    return document.getElementById(this.anchor ?? '');
```

Comment:
Should this be `this.getRootNode()` instead of `document`?

Reply:
Had to couch it a bit to make TS happy...

```
const rootNode = this.getRootNode();
return (rootNode instanceof ShadowRoot ? rootNode : document).getElementById(this.anchor ?? '');
```

But this is in and consistent with other uses in the package. It was saying `Node` doesn't not have method `getElementById`, but checking for `instanceof ShadowRoot` seems to pacify that. Open to more rounds of revision here if necessary.